src/Frontend/Modules/Blog/Engine/Model.php

```diff
@@ -47,7 +47,7 @@ public static function get($URL)
              INNER JOIN meta AS m2 ON c.meta_id = m2.id
              WHERE i.status = ? AND i.language = ? AND i.hidden = ? AND i.publish_on <= ? AND m.url = ?
              LIMIT 1',
-            array('active', LANGUAGE, 'N', FrontendModel::getUTCDate('Y-m-d H:i') . ':00', (string) $URL)
+            array('active', LANGUAGE, 'N', FrontendModel::getUTCDate('Y-m-d H:i'), (string) $URL)
```

Comment:
very well, just fix the merge conflicts and you are on your way to becoming a fork master!